### DISCUSSION TITLE
Picker -> Add extra props

### DISCUSSION THREAD

**Discussion by ncortesm** [wix/react-native-ui-lib#1772] [Ideas]
Is there a possibility to add extra properties to picker? for example

```
<Picker>
    {Array.map(val => (
      <Picker.Item key={val.value} value={val.value} label={val.label} extraProps={{"prop1": prop1, "prop2": prop2}}/>
    ))}
</Picker>
```

**Comment by ethanshar** [comment]
Hi @ncortesm 
Can you elaborate on why do you need to pass extra props to the Picker.Item? 
How would you use them?

**Reply by ncortesm** [reply]
Hi @ethanshar sorry for the delay,

my idea is to provide the possibility of dynamically changing the content of the DOM, by selecting a certain value.
a basic example:

we have an object that specifies a group of cars
```
const cars = [
  {label: 'Car1', value: 'car1', extraProps={"color": "blue"} },
  {label: 'Car2', value: 'car2', extraProps={"color": "red"}},
];
```

```
<Picker>
    {cars.map(val => (
      <Picker.Item key={val.value} value={val.value} label={val.label} extraProps={val.extraProps}/>
    ))}
</Picker>
```

When the user selects a blue car, the software changes the style of a component...
